R/print.R

```diff
@@ -71,10 +71,8 @@ print.gsplot <- function(x, ...){
     box()
   }
 
-  draw_legend(x)
-  draw_axis(x)
-
-  # par(defaultPar)
+  draw_legend(views)
+  draw_axis(views)
   
 }
 
```

Comment:
I can't seem to write a comment next to a picture...but it's interesting how the ticks are behaving now. The top ones cover the legend, the left don't